etl/steps/data/garden/wb/2023-07-24/food_prices_for_nutrition.countries.json

```diff
@@ -0,0 +1,188 @@
+  "GBR": "United Kingdom",
+  "GHA": "Ghana",
+  "GIN": "Guinea",
+  "GMB": "Gambia",
+  "GNB": "Guinea-Bissau",
+  "GNQ": "Equatorial Guinea",
+  "GRC": "Greece",
+  "GRD": "Grenada",
+  "GUY": "Guyana",
+  "HKG": "Hong Kong",
+  "HND": "Honduras",
+  "HRV": "Croatia",
+  "HTI": "Haiti",
+  "HUN": "Hungary",
+  "IDN": "Indonesia",
+  "IND": "India",
+  "IRL": "Ireland",
+  "IRN": "Iran",
+  "IRQ": "Iraq",
+  "ISL": "Iceland",
+  "ISR": "Israel",
+  "ITA": "Italy",
+  "JAM": "Jamaica",
+  "JOR": "Jordan",
+  "JPN": "Japan",
+  "KAZ": "Kazakhstan",
+  "KEN": "Kenya",
+  "KGZ": "Kyrgyzstan",
+  "KHM": "Cambodia",
+  "KNA": "Saint Kitts and Nevis",
+  "KOR": "South Korea",
+  "KWT": "Kuwait",
+  "LAO": "Laos",
+  "LBR": "Liberia",
+  "LCA": "Saint Lucia",
+  "LKA": "Sri Lanka",
+  "LSO": "Lesotho",
+  "LTU": "Lithuania",
+  "LUX": "Luxembourg",
+  "LVA": "Latvia",
+  "MAR": "Morocco",
+  "MDA": "Moldova",
+  "MDG": "Madagascar",
+  "MDV": "Maldives",
+  "MEX": "Mexico",
+  "MKD": "North Macedonia",
+  "MLI": "Mali",
+  "MLT": "Malta",
+  "MMR": "Myanmar",
+  "MNE": "Montenegro",
+  "MNG": "Mongolia",
+  "MOZ": "Mozambique",
+  "MRT": "Mauritania",
+  "MSR": "Montserrat",
+  "MUS": "Mauritius",
+  "MWI": "Malawi",
+  "MYS": "Malaysia",
+  "NAM": "Namibia",
+  "NER": "Niger",
+  "NGA": "Nigeria",
+  "NIC": "Nicaragua",
+  "NLD": "Netherlands",
+  "NOR": "Norway",
+  "NPL": "Nepal",
+  "NZL": "New Zealand",
+  "OMN": "Oman",
+  "PAK": "Pakistan",
+  "PAN": "Panama",
+  "PER": "Peru",
+  "PHL": "Philippines",
+  "POL": "Poland",
+  "PRT": "Portugal",
+  "PRY": "Paraguay",
+  "PSE": "Palestine",
+  "QAT": "Qatar",
+  "ROU": "Romania",
+  "RUS": "Russia",
+  "RWA": "Rwanda",
+  "SAU": "Saudi Arabia",
+  "SDN": "Sudan",
+  "SEN": "Senegal",
+  "SGP": "Singapore",
+  "SLE": "Sierra Leone",
+  "SLV": "El Salvador",
+  "SRB": "Serbia",
+  "STP": "Sao Tome and Principe",
+  "SUR": "Suriname",
+  "SVK": "Slovakia",
+  "SVN": "Slovenia",
+  "SWE": "Sweden",
+  "SWZ": "Eswatini",
+  "SXM": "Sint Maarten (Dutch part)",
+  "SYC": "Seychelles",
+  "TCA": "Turks and Caicos Islands",
+  "TCD": "Chad",
+  "TGO": "Togo",
+  "THA": "Thailand",
+  "TJK": "Tajikistan",
+  "TTO": "Trinidad and Tobago",
+  "TUN": "Tunisia",
+  "TUR": "Turkey",
+  "TWN": "Taiwan",
+  "TZA": "Tanzania",
+  "UGA": "Uganda",
+  "URY": "Uruguay",
+  "USA": "United States",
+  "VCT": "Saint Vincent and the Grenadines",
+  "VGB": "British Virgin Islands",
+  "VNM": "Vietnam",
+  "ZAF": "South Africa",
+  "ZMB": "Zambia",
+  "ZWE": "Zimbabwe",
+  "BON": "Bonaire (WB)",
+  "EAS": "East Asia & Pacific (WB)",
+  "ECS": "Europe & Central Asia (WB)",
+  "HIC": "High-income countries",
+  "LCN": "Latin America & Caribbean (WB)",
+  "LIC": "Low-income countries (WB)",
+  "LMC": "Lower-middle-income countries",
+  "MEA": "Middle East & North Africa (WB)",
+  "NAC": "North America (WB)",
+  "SAS": "South Asia (WB)",
+  "SSF": "Sub-Saharan Africa (WB)",
+  "UMC": "Upper-middle-income countries",
```

Comment:
Good catch! By the way, I'm assuming that their income groups are consistent (and up-to-date) in all World Bank datasets (and hence consistent with OWID), let me know if that's not a good assumption.

Reply:
I assume that as well, but it will depend on when that dataset was released. If it's sufficiently old it can have a previous definition of income groups. [I see they change every year](https://owid.cloud/admin/charts/1257/edit).

Reply:
Now that you say it, it should be that groups are dynamically changing per year, so that would be consistent. I don't know that with 100% certainty.

Reply:
If they assume a definition of income groups that changes over the years, then those income groups are inconsistent with ours. Because we assume the **current** definition of income groups. But for this dataset (with just a few years of data) I suppose it should not matter much.

Reply:
Right, I wasn't sure about that. I would assume theirs is dynamic, but I don't know